web/themes/joinup/joinup_theme.theme

```diff
@@ -1548,7 +1548,8 @@ function joinup_theme_preprocess_rdf_entity__licence__search_list_item(&$variabl
 
     $variables['spdx_licence'] = $spdx_licence;
     $variables['spdx_title'] = $variables['content']['spdx_licence_label'];
-    $spdx_url = Url::fromUri($spdx_licence->id(), ['fragment' => 'licenceText']);
+    $url = preg_replace('/(.*)\.html$/', '\1', $spdx_licence->id()) . '.html';
```

Comment:
This is application logic which does not belong in the theme layer. We should be able to switch to a different theme and still have correct links rendered.

Reply:
We can do this by providing a template override in `joinup_licence.module`.